masonry/src/contexts.rs

```diff
@@ -252,16 +252,22 @@ impl_context_method!(
         /// mouse position have hot status.
         ///
         /// Discussion: there is currently some confusion about whether a
-        /// widget can be considered hot when some other widget is active (for
-        /// example, when clicking to one widget and dragging to the next).
-        /// The documentation should clearly state the resolution.
+        /// widget can be considered hot when some other widget has captured the
+        /// pointer (for example, when clicking one widget and dragging to the
+        /// next). The documentation should clearly state the resolution.
         pub fn is_hot(&self) -> bool {
             self.widget_state.is_hot
         }
 
-        // TODO - remove
-        pub fn is_active(&self) -> bool {
-            self.global_state.pointer_capture_target == Some(self.widget_id())
+        /// Whether the pointer is captured by this widget.
+        ///
+        /// A widget can capture the pointer by calling [`capture_pointer`] during a pointer event.
+        /// It can subsequently release it by calling [`release_pointer`] during a pointer event.
```

Comment:
These docs now don't match the other new docs. Maybe we should just point at `capture_pointer`:
```rust
/// Any widget can [`release`] the pointer during any event. The pointer is automatically
/// released after handling of a [`PointerUp`] or [`PointerLeave`] event completes. A widget
/// holding the pointer capture will be the target of these events.
```

That is, the docs here imply that `release_pointer` is never implicit (and that it can only be called during a pointer event). I believe that neither of those are true.

Reply:
Good idea. That also means fewer docs to keep in sync.